sd_jwt/bin/sd_jwt

```diff
@@ -23,325 +22,235 @@ logger = logging.getLogger("sd_jwt")
+    ),
+    type=pathlib.Path,
 )
 parser.add_argument(
-    '-d', '--debug', required=False,
-    choices=(
-        'CRITICAL',
-        'ERROR',
-        'WARNING',
-        'INFO',
-        'DEBUG'
-    ),
-    default='INFO',
-    help="Debug level, see python logging; defaults to INFO if omitted"
+    "-d",
+    "--debug",
+    required=False,
+    choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"),
+    default="INFO",
+    help="Debug level, see python logging; defaults to INFO if omitted",
 )
 parser.add_argument(
-    '-nr', '--no-randomness', required=False,
+    "-nr",
+    "--no-randomness",
+    required=False,
     action="store_true",
     default=False,
     help=(
         "For the purpose of generating static examples for the spec, this command line "
         "switch disables randomness. Using this in production is highly insecure!"
-    )
+    ),
 )
 parser.add_argument(
-    '--nonce', required=False,
+    "--nonce",
+    required=False,
     type=str,
     default=generate_salt(),
-    help=(
-        "given example of a salt: 'XZOUco1u_gEPknxS78sWWg'"
-    )
+    help=("given example of a salt: 'XZOUco1u_gEPknxS78sWWg'"),
 )
 parser.add_argument(
-    '--iat', required=False,
-    type=int,
-    help=("issued at, UTC Timestamp")
+    "--iat", required=False, type=int, help=("issued at, UTC Timestamp")
 )
 parser.add_argument(
-    '--exp', required=False,
-    type=int,
-    help=("expire at, UTC Timestamp")
+    "--exp", required=False, type=int, help=("expire at, UTC Timestamp")
 )
 parser.add_argument(
-    '--settings-path', required=False,
+    "--settings-path",
+    required=False,
     type=str,
-    help=(
-        "path/to/settings.py"
-        ". This path/to requires a __init__.py file."
-    )
+    help=("path/to/settings.py - this path/to requires a __init__.py file."),
 )
 parser.add_argument(
-    '--indent', required=False,
+    "--indent",
+    required=False,
     type=int,
     default=4,
-    help=("json output indentation level")
+    help=("json output indentation level"),
 )
 parser.add_argument(
-    '--replace-examples-in', required=False,
-    type=str,
+    "--replace-examples-in",
+    required=False,
+    type=pathlib.Path,
+    default=pathlib.Path("main.md"),
     help=(
-        "path/to/main.md"
-        " - replaces all the examples in the specs"
-    )
+        "path/to/main.md - Do not output examples, but replace all the examples in the specs"
+    ),
 )
 parser.add_argument(
-    '-v', '--version', required=False,
+    "-v",
+    "--version",
+    required=False,
     action="store_true",
-    help="Print version and exit"
+    help="Print version and exit",
 )
 
 _args = parser.parse_args()
 logger.setLevel(_args.debug)
 
 if _args.version:
-    sys.exit(f'{__version__}')
+    sys.exit(f"{__version__}")
+
+### Load settings
 
-# load settings
 if _args.settings_path:
     sys.path.append(_args.settings_path)
-    from settings import *
+    from settings import (
```

Comment:
* prevented code repetition but this solution is fair

Reply:
Keeping as proposed is okay?